flink-next-gen-connector-influxdb/src/main/java/org/apache/flink/streaming/connectors/influxdb/sink/InfluxDBCommittableSerializer.java

```diff
@@ -33,12 +33,13 @@ public int getVersion() {
     }
 
     @Override
-    public byte[] serialize(final Void obj) throws IOException {
+    public byte[] serialize(final Long obj) {
         return SimpleVersionedStringSerializer.INSTANCE.serialize(String.valueOf(obj));
     }
 
     @Override
-    public Void deserialize(final int version, final byte[] serialized) throws IOException {
-        return null;
+    public Long deserialize(final int version, final byte[] serialized) throws IOException {
+        return Long.valueOf(
+                SimpleVersionedStringSerializer.INSTANCE.deserialize(version, serialized));
```

Comment:
Reimplement the SimpleVersionedStringSerializer to a SimpleVersionedLongSerializer. ATM, it does not make sense...